scripts/docker/DSS.Dockerfile

```diff
@@ -4,47 +4,59 @@ COPY dss-ansible/artifacts/dss-gcc510-*.rpm ./
 COPY dss-ansible/artifacts/aws-sdk-cpp-*.rpm ./
 RUN set -eux \
 	&& yum install -y \
-        epel-release centos-release-scl-rh && \
-    yum group install -y \
-        "Development Tools" && \
+        epel-release \
+        centos-release-scl-rh && \
     yum install -y \
 		bc \
-        boost-devel \
-        check \
-        cmake \
-        cmake3 \
-        CUnit-devel \
-        dejagnu \
-        devtoolset-11 \
-        dpkg \
-        elfutils-libelf-devel \
-        expect \
-        glibc-devel \
-        jemalloc-devel \
-        Judy-devel \
-        libaio-devel \
-        libcurl-devel \
-        libuuid-devel \
-        meson \
-        ncurses-devel \
-        numactl-devel \
-        openssl-devel \
-        pulseaudio-libs-devel \
-        python3 \
-        python3-devel \
-        python3-pip \
-        rdma-core-devel \
-        redhat-lsb \
-        ruby-devel \
-        snappy-devel \
-        tbb-devel \
-        wget \
-        zlib-devel \
+        bison \
+		boost-devel \
+		cmake \
```

Comment:
Hope this indentation is fine on your end

Reply:
Good catch. This is now fixed.